src/appReview/slice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export interface State {
+  lastInteractionTimestamp: number | null
+}
+
+const initialState: State = {
+  lastInteractionTimestamp: null,
+}
+
+export const slice = createSlice({
+  name: 'appReview',
+  initialState,
+  reducers: {
+    setLastInteractionTimestamp: (state, action) => {
```

Comment:
I know the codebase is not following this in a bunch of places, but it's better to model actions as events instead of setters.
https://redux.js.org/style-guide/#model-actions-as-events-not-setters

Maybe?
```suggestion
   inAppReviewShown: (state, action) => {
```

Reply:
Fixed in commits/cfda34c959aa53f7095cdd15007b69e8b72be60e!